hail/src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -67,14 +67,12 @@ object VEP {
     }
   }
 
-  def waitFor(proc: Process, cmd: Array[String]): Unit = {
+  def waitFor(proc: Process, err: StringBuilder, cmd: Array[String]): Unit = {
     val rc = proc.waitFor()
 
     if (rc != 0) {
-      val errorLines = Source.fromInputStream(new BufferedInputStream(proc.getErrorStream)).getLines().mkString("\n")
-
       fatal(s"VEP command '${ cmd.mkString(" ") }' failed with non-zero exit status $rc\n" +
-        "  VEP Error output:\n" + errorLines)
+        "  VEP Error output:\n" + err.toString)
```

Comment:
Can you help me understand why this works when `proc.getErrorStream` doesn't? I expected we would have an err stream and an out stream and we would include both (separately) in the exception message.

Reply:
The output stream is VEP's output, which we parse. The issue here is with the `pipe` method, changed later in this PR. We set up a thread to read from VEP's stderr pipe. We can only read from that stream once. Previously we were printing it to the JVM's `System.err` stream, now, we're capturing it in a `StringBuilder`.